### DISCUSSION TITLE
Prima installazione HA OS su miniPC

### DISCUSSION THREAD

**Discussion by Piubeni** [indomus/forum#3069] [HUB: Home Assistant]
Ho installato per la prima volta ( sono neofita...) HA OS su miniPC (intel) seguendo la procedura prevista senza alcun problema compreso copia di OS su disco del mini PC. 
Quando alla fine riavvio il miniPC mi compare la scritta "Reboot and Select proper Boot device or Insert Boot Media in selected Boot device and press key". 
Il mini PC è connesso a internet con rete cablata verificata.
Nel BIOS ci sono n. 3 opzioni boot messe in questo ordine:
WDC PC SN520 SDAPNUW-128G-1006
UEFI : PXE IP4 Realtek PCIe GBE Family Controller
UEFI : PXE IP6 Realtek PCIe GBE Family Controller
Cosa posso fare grazie?

**Comment by Piubeni** [comment]
Ho provato anche a invertire l'ordine di caricamento (prima UEFI : PXE IP4 ....) e mi compare:
">> Checking Media Presence....
>> Media present .....
>> Start PXE over IPv4."
Poi stesso testo ma per IPv6,
e poi la scritta "Reboot and Select ....."
Grazie ancora ...

**Comment by KC-inDomus** [comment]
> Ho installato per la prima volta ( sono neofita...) HA OS su miniPC (intel) seguendo la procedura prevista senza alcun problema compreso copia di OS su disco del mini PC. Quando alla fine riavvio il miniPC mi compare la scritta "Reboot and Select proper Boot device or Insert Boot Media in selected Boot device and press key". Il mini PC è connesso a internet con rete cablata verificata. Nel BIOS ci sono n. 3 opzioni boot messe in questo ordine: WDC PC SN520 SDAPNUW-128G-1006 UEFI : PXE IP4 Realtek PCIe GBE Family Controller UEFI : PXE IP6 Realtek PCIe GBE Family Controller Cosa posso fare grazie?

hai disattivato il safe boot?

**Reply by Piubeni** [reply]
Si ho disattivato il safe boot già prima dell'installazione...

**Reply by KC-inDomus** [reply]
SE non hai safe boot SE hai installato correttamente l'immagine SE l'ordine di boot è corretto, DEVE partire. il che mi porta a pensare che hai commesso qualche errore :) ribatti tutto.

**Comment by sdavides** [comment]
Leggo delle note riguardo al boot per alcuni bios

https://www.home-assistant.io/installation/generic-x86-64#start-up-your-generic-x86-64

Forse è questo?

**Reply by Piubeni** [reply]
Grazie sdavides sei stato la mia salvezza!!! 
Infatti ho provato  ripetutamente ad installare secondo la procedura prevista HA OS pensando di aver commesso qualche errore, ma senza alcun risultato. Ho successivamente seguito le indicazioni del link che hai postato ed ho  caricato la riga di comando raccomandata per rendere compatibile la BIOS  :
efibootmgr --create --disk /dev/<drivename> --part 1 --label "HAOS" \   --loader '\EFI\BOOT\bootx64.efi'
Dopo il riavvio il miniPC ha finalmente cominciato l'aggiornamento e tutto funziona!!!
grazie ancora

**Reply by KC-inDomus** [reply]
grazie anche a te, ci hai fatto indirettamente scoprire una cosa nuova - che mettiamo in guida. grazie!
